### PR TITLE
Adds steps for enabling TLS 1.3

### PR DESCRIPTION
**Pull request by brewn**

### CODE REVIEW

**Comment by brewn** [comment]
#629

**Review by juliandunn — CHANGES_REQUESTED** [review]
Authentication protocol 1.3 is *not* a TLS version; it refers to the custom (Chef-developed) protocol that clients and servers use to sign their requests.

File: chef_master/source/config_rb.rst [hunk 1]
```diff
@@ -73,6 +80,11 @@ This configuration file has the following settings:
 
       data_bag_encrypt_version 2
 
+``fips``
+  Allows OpenSSL to enforce FIPS-validated security during the chef-client run. Set to ``true`` to enable FIPS-validated security.
+
+  Changed in Chef server 12.13 to expose FIPS runtime flag on RHEL.
```
Comment (juliandunn) [inline chef_master/source/config_rb.rst]: I don't know what that means

Comment (brewn) [reply chef_master/source/config_rb.rst]: This was pulled in as part of a replication between the settings in the knife.rb documentation and those in config.rb, as the two were beginning to splinter, with knife.rb getting more updates. 

The "Changed in Chef server 12.13 to expose FIPS runtime flag on RHEL." line does seem like a non-sequitur here, though. I think it's meant to be a very roundabout way of saying that FIPS mode is only supported on RHEL for Chef server.

While the [notes](https://github.com/chef/chef-server/commit/4160a18946b07b9559786abfb32a1976f52008ca) say the same thing, I suspect that if FIPS is supported for server on RHEL it likely works on CentOS and Oracle as well..

**Edit:** Confirmed platform support with @tyler-ball

**Pull request merged**